### PR TITLE
leaflet-CanvasMarker plugin

### PR DESCRIPTION
**Pull request by liubingreat**

### CODE REVIEW

**Comment by liubingreat** [comment]
add a leaflet plugin, display markers on canvas.

File: docs/plugins.md [hunk 1]
```diff
@@ -1845,6 +1845,19 @@ These plugins provide new markers or news ways of converting abstract data into
...
 		</td>
 	</tr>
+	<tr>
+		<td>
+			<a href="https://github.com/liubingreat/leaflet-CanvasMarker">leaflet-CanvasMarker</a>
+		</td>
+		<td>
+			A Leaflet plugin display markers and labels on canvas.
+			(<a href="http://180.76.171.45:8080/leaflet-canvas-marker/example/demo.html">demo</a>).
+			(<a href="http://180.76.171.45:8080/leaflet-canvas-marker/example/label-demo.html">label-demo</a>).
```
Comment (johnd0e) [inline docs/plugins.md:1855]: ERR_CONNECTION_TIMED_OUT

**Review by johnd0e — CHANGES_REQUESTED** [review]

**Comment by johnd0e** [comment]
@liubingreat 
Please resolve conflicts

**Comment by Falke-Design** [comment]
We changed the design / structur for the plugins site. Now each plugin have a own file. Please reaname your plugin and then create a new PR #7805

**Pull request closed without merging**